### DISCUSSION TITLE
How to Localize text returned by ViewClipDecoration. getDecorationToolTip

### DISCUSSION THREAD

**Discussion by muthubentley** [iTwin/itwinjs-core#5216] [Q&A] [answered]
Our Partner uses ViewClipDecoration in section functionality. They show the text returned from ViewClipDecoration. getDecorationToolTip in the UI. The text is in English and partner wants to know if its possible to localize this text
Function declaration:
core/frontend/src/tools/ClipViewTool.ts
async getDecorationToolTip(hit) { return (hit.sourceId === this._clipId ? "View Clip" : "Modify View Clip"); }

![image](https://user-images.githubusercontent.com/107229931/225510401-f5a1d5a1-090a-42f2-8f4c-c7b9eeab13f0.png)

**Comment by pmconne** [comment, marked as answer]
@bbastings

**Reply by bbastings** [reply]
Missing localization [fix](https://github.com/iTwin/itwinjs-core/commit/8fd0b43845a8abfb73b003f9f4cf007a315a71cb).

**Reply by muthubentley** [reply]
@bbastings  when this fix will be released ?

**Reply by pmconne** [reply]
@muthubentley it is currently slated for 4.0. We can backport to 3.7 if necessary. If it's urgent for some reason, we can backport to 3.6. You tell us.

**Reply by muthubentley** [reply]
@pmconne Partner is planning to wrap up development by this Month. So a release in 3.6 or 3.7 would be very helpful for them.

**Reply by pmconne** [reply]
#5244
